README.md

```diff
@@ -488,6 +488,28 @@ and the `GitHub` theme when in the _light mode_.
 alias cat="bat --theme=\$(defaults read -globalDomain AppleInterfaceStyle &> /dev/null && echo default || echo GitHub)"
 ```
 
+### Syntax of a file
+
+You can change the default syntax highlighting of the file using the `--map-syntax` option.
+It uses glob pattern matching and is matched on full path and filename.
```

Comment:
"It supports" instead of "it uses"

This one is more of a pedantic nitpick (and completely optional), but I feel like it reads better. The option does use globs, but saying it supports them is more accurate since it doesn't have the implication of `uses -> relies on` behind it.

Reply:
Not pedantic at all. It sounds better.